build.gradle

```diff
@@ -18,10 +18,11 @@ allprojects {
         options.fork = true  // Fork your compilation into a child process
         options.forkOptions.setMemoryMaximumSize("4g") // Set maximum memory to 4g
         options.compilerArgs << "-Xlint:all"
-        options.compilerArgs << "-Werror"
+        // options.compilerArgs << "-Werror"
```

Comment:
@pfend, @kordianbruck: This are the actual warnings I get:
```
Warning:No processor claimed any of these annotations: java.lang.SafeVarargs,android.databinding.BindingBuildInfo,org.simpleframework.xml.ElementList,android.support.annotation.NonNull,com.google.gson.annotations.SerializedName,org.simpleframework.xml.Root,retrofit2.http.PUT,android.annotation.SuppressLint,retrofit2.http.DELETE,retrofit2.http.GET,retrofit2.http.Path,org.simpleframework.xml.Text,android.support.annotation.Nullable,retrofit2.http.POST,retrofit2.http.Body,org.simpleframework.xml.Element,android.annotation.TargetApi,android.databinding.Bindable

/Users/sebastianstein/Google Drive/Uni/android praktikum/TumCampusApp/app/build/generated/source/apt/debug/de/tum/in/tumcampusapp/databinding/ActivityCardsQuizBinding.java
Warning:(59, 37) [cast] redundant cast to String
Warning:(85, 36) [cast] redundant cast to String
Warning:(180, 112) [unchecked] unchecked cast
required: ObservableField<Boolean>
found:    Object

/Users/sebastianstein/Google Drive/Uni/android praktikum/TumCampusApp/app/build/generated/source/apt/debug/de/tum/in/tumcampusapp/databinding/ActivityCardsDetailBinding.java
Warning:(52, 32) [cast] redundant cast to String
Warning:(78, 37) [cast] redundant cast to String
Warning:(104, 36) [cast] redundant cast to String
```
The first one appears since i enabled databinding for the project.
The others are cast warnings in the Binding Classes which are generated by the databinding library.
So I cant simply fix the cast problems there or add the `@SuppressWarnings` annotation.

But I totally understand why this flag is set and i am going to investigate on this after the "hot time" of the praktikum.